lib/connectors/memory.js

```diff
@@ -823,7 +824,7 @@ Memory.prototype.replaceOrCreate = function(model, data, options, callback) {
     // Calling _createSync to update the collection in a sync way and
     // to guarantee to create it in the same turn of even loop
     return self._createSync(model, data, function(err, id) {
-      if (err) return process.nextTick(function() { cb(err); });
+      if (err) return process.nextTick(function() { callback(err); });
```

Comment:
It's a nitpick anyways, 2 commits (one for bug and one for the eslint changes) would've been better for the reviewer to determine WHY the cb/callback thing was changed since the bugfix is not a requirement of ESLint since this PR was for ESLint changes. I'm not saying make a separate PR, just another commit on top of the current changes.